tests/test_basics.py

```diff
@@ -4,7 +4,7 @@
 
 import datetime
 
-from cache import tax_benefit_system
+from .cache import tax_benefit_system
```

Comment:
Absolute imports :)

Reply:
The `test` package is not under `openfisca_france`, so I don't see how I can use an absolute import.

Reply:
`tests.cache` ?